Knossos.NET/ViewModels/Templates/DebugFiltersViewModel.cs

```diff
@@ -64,95 +64,81 @@ public async Task LoadDebugRepo()
                 }
             }
 
-            // Load info from any valid config
-            bool configFound = false;
-
-            foreach(var mod in Knossos.GetInstalledModList(null))
-            {
-                try {
-                    // Debug filter files are always in the data folder.
-                    var path = Path.Combine(mod.fullPath, "data", "debug_filter.cfg");
-                    if (File.Exists(path)) 
+            try {
+                // Debug filter files are always in the data folder.
+                var path = Path.Combine(KnUtils.GetFSODataFolderPath(), "data", "debug_filter.cfg");
+                if (File.Exists(path)) 
+                {
+                    using (StreamReader reader = new StreamReader(Path.Combine(KnUtils.GetFSODataFolderPath(), "data", "debug_filter.cfg"))) 
```

Comment:
Can't you just reuse the `path` variable here instead?

Reply:
Yes, thanks.  I didn't think about it when I removed the loop.